CliFx/Schema/CommandSchema.cs

```diff
@@ -86,13 +86,19 @@ internal partial class CommandSchema
         var implicitOptionSchemas = string.IsNullOrWhiteSpace(name)
             ? new[] {OptionSchema.HelpOption, OptionSchema.VersionOption}
             : new[] {OptionSchema.HelpOption};
-
-        var parameterSchemas = type.GetProperties()
+        
+        // Include interface members for multiple inheritance
+        // If interface inherits from ICommand, it will be included
+        var iType = typeof(ICommand);
+        var interfaces = type.GetInterfaces().Where(i => i != iType && iType.IsAssignableFrom(i) );
```

Comment:
This probably looks tidier since `typeof(...)` is cached anyway
```suggestion
        var interfaces = type.GetInterfaces().Where(i => i != typeof(ICommand) && typeof(ICommand).IsAssignableFrom(i));
```